graphene_plugin.py

```diff
@@ -545,12 +545,22 @@ class ObjectTypeInfo(BaseObjectInfo):
     * name: the name of the `ObjectType`
     * fields: A `dict` of field name to `FieldInfo` for each field defintion in the `ObjectType`
       (e.g. `some_field = Field(String, required=True, description='this is a thing')`)
+    * interface_fields: A `dict` of field name to `FieldInfo` for each field definition in all of the `ObjectType`'s
+      interfaces.
+    * all_fields: A `dict` of field name to `FieldInfo` for each field definition in the `ObjectType` and all of its
+      interfaces.
```

Comment:
First, I decided to untangle the data model a bit. I decided it should be a little more "honest" about what's actually happening in the code. So rather than actually copying the `Interface` `Field()`s here, I decided to separate them into `fields` (just the `Field()`s actually defined on the `ObjectType`,) `interface_fields` (the `Field()`s defined on the `Interface`s,) and `all_fields` (a combo of the two.)